### ISSUE TITLE
Move demo_zips files to Google Drive

### ISSUE THREAD

**Issue by davestanley** [DynaSim/DynaSim#215]
Using DynaSim Google Drive account to store zip files (DynaSimDev@gmail.com); removing these from repo

**Comment by davestanley** [comment]
Done. 
See dsDownloadFiles function.
Also, we are using github to host the files, not Google Drive

**Comment by davestanley** [comment]
**To summarize:**
Now there is a command dsDownloadFiles which we can use to automatically pull large files off of the web, as opposed to keeping them mixed in with the DynaSim code.

Basically, the files are hosted in a special branch of the DynaSim repo called dsFiles (this seemed easier than doing it through Google Drive).
https://github.com/DynaSim/DynaSim/tree/dsFiles

The command dsDownloadFiles will download files from a specific location within the dsFiles directory tree and save them to a corresponding location on your local copy of DynaSim.

In this way, the dsFiles branch can mirror our desired directory structure for the main DynaSim branches, without requiring us to actually store large files in these branches.

The downside of this, of course, is that it now means the users will need to have internet access in order to run certain DynaSim demo scripts.

**Comment by davestanley** [comment]
Update:

I deleted the dsFiles branch, which we were previously using to host DynaSim demo zip files. I've moved the contents of this branch to a new repo: https://github.com/DynaSim/DynaSimFiles

I also updated the command dsDownloadFiles to point to this new repo. All of the DynaSim developers should have received invites as collaborators to this repo.

The DynaSim repo still contains these zip files in certain other branches (e.g. master), even though they're now gone from dev, which is why the repo is still of substantial size:

```
du -sh DynaSim
 38M	DynaSim
```

This might go down on it's own once we update master or, otherwise, we can do a git filter to obliterate these files entirely.
